grants/caspershops/m4.md

```diff
@@ -0,0 +1,21 @@
+---
+Milestone: "M1",
+Status: "In review",
+---
+<!--lang:en--> 
+### Description
+
+Source code behind the contract that mints “X” amount of NFT’s based on the defined inventory to be listed.
+
+### Acceptance Criteria
+
+Deploy the initial contract onto Casper testnet
+
+### Progress Update
+
+**Reporting Period: 04.11.2022 - 31.01.2023**
+
+- Assessment of the operating system done
+- Environment preparation
+- Please refer to the Milestone review in the link [here](https://casper-association.atlassian.net/wiki/spaces/DropLinked/pages/284229635/Milestone+1)
```

Comment:
The Milestone review link reads "Milestone 1". Would you mind verifying, please?